W0613 17:37:02.316403 1557409 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 90.0, Driver API Version: 50400.0, Runtime API Version: 50400.0
Sycl_Compiler
I0613 17:37:02.432536 1557458 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_0.cc -o ./source/sycl_0.so
I0613 17:37:09.807644 1557409 sycl_module.cc:36] destructor for SYCLModule
.Sycl_Compiler
I0613 17:37:09.815158 1557509 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_1.cc -o ./source/sycl_1.so
I0613 17:37:17.175856 1557409 sycl_module.cc:36] destructor for SYCLModule
.Sycl_Compiler
I0613 17:37:17.181247 1557559 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_2.cc -o ./source/sycl_2.so
I0613 17:37:24.590214 1557409 sycl_module.cc:36] destructor for SYCLModule
.Sycl_Compiler
I0613 17:37:24.595885 1557609 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_3.cc -o ./source/sycl_3.so
I0613 17:37:32.628057 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:37:32.645160 1557659 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_4.cc -o ./source/sycl_4.so
I0613 17:37:40.078311 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:37:40.092752 1557709 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_5.cc -o ./source/sycl_5.so
I0613 17:37:47.526559 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:37:47.551515 1557759 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_6.cc -o ./source/sycl_6.so
I0613 17:37:55.687094 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:37:56.143478 1557809 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_7.cc -o ./source/sycl_7.so
I0613 17:38:03.589201 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:03.916500 1557859 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_8.cc -o ./source/sycl_8.so
I0613 17:38:11.368794 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:11.690409 1557909 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_9.cc -o ./source/sycl_9.so
I0613 17:38:19.061635 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:19.693804 1557959 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_10.cc -o ./source/sycl_10.so
I0613 17:38:27.156432 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:27.782541 1558009 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_11.cc -o ./source/sycl_11.so
I0613 17:38:35.845507 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:36.482388 1558059 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_12.cc -o ./source/sycl_12.so
I0613 17:38:43.903533 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:43.922052 1558109 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_13.cc -o ./source/sycl_13.so
I0613 17:38:52.023058 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:52.044659 1558159 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_14.cc -o ./source/sycl_14.so
I0613 17:38:59.485615 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:38:59.511255 1558209 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_15.cc -o ./source/sycl_15.so
I0613 17:39:06.877465 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:39:06.902693 1558259 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_16.cc -o ./source/sycl_16.so
I0613 17:39:14.935804 1557409 sycl_module.cc:36] destructor for SYCLModule
FSycl_Compiler
I0613 17:39:14.958168 1558309 compiler_sycl.cc:78] compile command: /opt/DPC++/bin/clang++ -I /workspace/Paddle/build/python/paddle/libs -fsycl -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906 -std=c++17 -O3 -fPIC -shared -ldl -fbracket-depth=1030 ./source/sycl_17.cc -o ./source/sycl_17.so
I0613 17:39:22.908195 1557409 sycl_module.cc:36] destructor for SYCLModule
F
======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase3)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(1024,), -6.911590e-01 vs -2.332392e-01. Maximum diff of the whole array: maximum_relative_diff=3.769512e+03, maximum_absolute_diff=3.988730e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase4)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(1024,), -1.138763e+00 vs -3.848813e-02. Maximum diff of the whole array: maximum_relative_diff=1.200352e+02, maximum_absolute_diff=7.435267e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase5)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(1024,), -9.931582e-01 vs -7.573691e-02. Maximum diff of the whole array: maximum_relative_diff=3.783422e+02, maximum_absolute_diff=1.995245e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase6)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 24420 different results, offset=8, shape=(512, 256), -1.468746e-01 vs -5.786133e-01. Maximum diff of the whole array: maximum_relative_diff=9.597203e+04, maximum_absolute_diff=1.278202e+01.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase7)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 24456 different results, offset=8, shape=(512, 256), -2.124869e+00 vs -5.938635e-01. Maximum diff of the whole array: maximum_relative_diff=4.173109e+04, maximum_absolute_diff=1.473451e+01.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase8)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 24420 different results, offset=8, shape=(512, 256), -1.249239e-01 vs -1.044024e+00. Maximum diff of the whole array: maximum_relative_diff=2.702261e+04, maximum_absolute_diff=3.568915e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase9)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 49044 different results, offset=8, shape=(128, 64, 32), -4.099320e-01 vs -5.288756e-01. Maximum diff of the whole array: maximum_relative_diff=3.745276e+05, maximum_absolute_diff=1.037906e+01.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase10)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 48924 different results, offset=8, shape=(128, 64, 32), -6.137674e-01 vs -1.024194e+00. Maximum diff of the whole array: maximum_relative_diff=1.854638e+04, maximum_absolute_diff=1.546880e+01.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase11)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 48996 different results, offset=8, shape=(128, 64, 32), -3.794472e-02 vs -1.637704e-01. Maximum diff of the whole array: maximum_relative_diff=1.485174e+04, maximum_absolute_diff=5.065358e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase12)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(16, 8, 4, 2), -1.869978e+00 vs -1.103469e+00. Maximum diff of the whole array: maximum_relative_diff=1.295988e+02, maximum_absolute_diff=4.729835e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase13)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(16, 8, 4, 2), -4.411456e+00 vs -2.603400e+00. Maximum diff of the whole array: maximum_relative_diff=1.799787e+02, maximum_absolute_diff=6.352361e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase14)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(16, 8, 4, 2), -1.003502e-01 vs -4.724204e-01. Maximum diff of the whole array: maximum_relative_diff=2.463161e+02, maximum_absolute_diff=2.024300e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase15)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(16, 8, 4, 2, 1), -1.919777e+00 vs -8.654613e-01. Maximum diff of the whole array: maximum_relative_diff=1.201612e+02, maximum_absolute_diff=5.759654e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase16)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(16, 8, 4, 2, 1), -4.253767e+00 vs -7.208607e+00. Maximum diff of the whole array: maximum_relative_diff=1.550480e+02, maximum_absolute_diff=6.877361e+00.

======================================================================
FAIL: test_check_results (op_test_helper.TestLogOpShape.TestLogeOpCase17)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_log_op.py", line 73, in test_check_results
    self.check_outputs_and_grads()
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 158, in check_outputs_and_grads
    self.check_results(
  File "/workspace/Paddle/test/cinn/ops/op_test.py", line 313, in check_results
    self.assertTrue(is_allclose, msg=error_message)
AssertionError: False is not true : [Check Outputs] [CPU] The 0-th output: total 192 different results, offset=8, shape=(16, 8, 4, 2, 1), -3.609893e-01 vs -2.663968e-01. Maximum diff of the whole array: maximum_relative_diff=4.995196e+02, maximum_absolute_diff=1.917874e+00.

----------------------------------------------------------------------
Ran 18 tests in 140.604s

FAILED (failures=15)
